version/version.go

```diff
@@ -4,5 +4,5 @@
 
 package version
 
-const defaultBeatVersion = "8.11.0"
+const defaultBeatVersion = "8.10.0"
```

Comment:
8.11.0 is the version to have on main

Reply:
I know, but while the 8.11 artefacts aren't all published, the buildkite build will fail because it cannot download the agent's components. Therefore I changed it to 8.10 so I can have the tests running.